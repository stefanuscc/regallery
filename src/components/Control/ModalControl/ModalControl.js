import styles from './ModalControl.module.scss'
import Button from '../../micro/Button/Button'

export const ModalControl = (props) => {
    const { className='', onSave, onCancel } = props

    return (
        <span className={`${styles.group} ${className}`}>
            <Button isIcon={false} isActive={true} onClick={onSave}>
                SAVE
            </Button>
            <Button isIcon={false} isActive={false} onClick={onCancel}>
                CANCEL
            </Button>
        </span>
    )
}

export default ModalControl