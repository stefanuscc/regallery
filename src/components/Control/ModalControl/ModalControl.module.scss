.group {
    display: flex;
    width: 100%;
    flex-direction: row;
    align-items: stretch;
    margin-top: .5rem;
    justify-content: space-between;
}