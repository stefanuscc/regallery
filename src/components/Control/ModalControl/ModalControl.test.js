import React from 'react';
import renderer from 'react-test-renderer';
import ModalControl from './ModalControl';
import "../../../setupTests"

test('Modal Control to match snapshot', () => {
    const component = renderer.create(
        <ModalControl onSave={() => {}} onCancel={() => {}}/>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});