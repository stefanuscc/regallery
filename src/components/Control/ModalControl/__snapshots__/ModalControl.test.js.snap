// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal Control to match snapshot 1`] = `
<span
  className="group "
>
  <button
    className="button  active "
    onClick={[Function]}
    type="button"
  >
    SAVE
  </button>
  <button
    className="button   "
    onClick={[Function]}
    type="button"
  >
    CANCEL
  </button>
</span>
`;
